test/RequestResponseTest.cpp

```diff
@@ -157,6 +157,23 @@ TEST(RequestResponseTest, RequestOnDisconnectedClient) {
   ASSERT(did_call_on_error);
 }
 
+// TODO: test that multiple requests on a requestResponse
+// fail in a well-defined way (right now it'd nullptr deref)
+TEST(RequestResponseTest, MultipleRequestsError) {
```

Comment:
oops, this should go in a separate commit; ignore this

Reply:
So is this being removed?